docs/2022/ui/updates/krishna/2022-07-04.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Week 3
+author: Krishna Mahato
+author_url: https://github.com/krishna9304
+last_updated: 4th July, 2022
+author_image_url: https://avatars.githubusercontent.com/u/71918441?v=4
+tags: [gsoc22, react, Week 3]
+---
+
+<!--
+SPDX-License-Identifier: CC-BY-SA-4.0
+
+SPDX-FileCopyrightText: 2022 Krishna Mahato <krishhtrishh9304@gmail.com>
+-->
+
+## Goals for the week
+
+| Sl. No. | Initial Goals                                                   | Completed   |
+| ------- | --------------------------------------------------------------- | ----------- |
+| 1.      | Resolve the requested changes by mentors in the existing PRs    | Done        |
+| 2.      | Settting up the fossology main repo devlopment environment      | In Progress |
+| 3.      | Getting started with the creation of REST API for adding a user | Not started |
+
+## Roadblocks / hurdles
+
+- Since, I am using a _**macOS**_ based system, As quoted by the mentors --- fossology isn't compatible with macOS, It has to be a linux distribution (ubuntu/fedora).
+- With lots of research, I finally decided to install a virtual machine. Most of the virtual machines weren't compatible with the `arm` architecture.
+- Finally I got one VM service which was supported in my system. It is called [**UTM**](https://mac.getutm.app/).
+- After setting up UBUNTU 22.04 on my **VM**, and installing all the dependencies via `sudo utils/fo-installdeps`, fossology still threw the following error,
+  <img width="729" alt="Screenshot 2022-07-04 at 11 23 07 AM" src="https://user-images.githubusercontent.com/71918441/177207047-02155d3d-0261-43be-b80f-08fc3d23a66e.png"/>
```

Comment:
Please upload the image to repo itself.

See PR #89 to understand how to do it.

Reply:
Okay sure. Done!